doc/source/openshift.md

```diff
@@ -8,19 +8,20 @@ You will need a running OpenShift cluster, or OKD cluster. You can provision [Op
 
 ## Steps
 
-1. [Create Watson services on IBM Cloud](#1-create-watson-services-on-ibm-cloud)
+1. [Create Watson services](#1-create-watson-services)
 1. [Create an OpenShift project](#2-create-an-openshift-project)
 1. [Create the config map](#3-create-the-config-map)
 1. [Run the web app](#4-run-the-web-app)
 
-## 1. Create Watson services on IBM Cloud
+## 1. Create Watson services
 
-Use the following links to create the Watson services on IBM Cloud. Copy/paste the `API Key` and `URL` or keep the browser tabs open. You'll need these later.
+Create the following services:
 
-* [**Watson Assistant**](https://cloud.ibm.com/catalog/services/conversation)
-* [**Watson Discovery**](https://cloud.ibm.com/catalog/services/discovery)
-* [**Watson Tone Analyzer**](https://cloud.ibm.com/catalog/services/tone-analyzer)
-* [**Watson Natural Language Understanding**](https://cloud.ibm.com/catalog/services/natural-language-understanding)
+* **Watson Assistant**
+* **Watson Discovery**
+* **Watson Natural Language Understanding**
+
+Use the instructions found for [Deploying Locally](local.md) to provision your services from IBM Cloud or from an IBM Cloud Pak for Data cluster. Once you have created the services and collected all of the service credential data, return to this page and complete the remaining steps.
```

Comment:
I think it is a bad idea to send then to local.md from here.  Either move the common stuff to the README (or some other common place) or copy it in here so they don't link to some wrong place.

Reply:
good idea - will create a new common section